pbft/pbft.hpp

```diff
@@ -249,29 +267,29 @@ namespace bzn
         enum class swarm_status {not_joined, joining, waiting, joined};
         swarm_status in_swarm = swarm_status::not_joined;
```

Comment:
shouldn't this be persisted? Or is the logic that we'll try to join and if we're already in it'll be handled appropriately

Reply:
There's currently no way to leave the swarm so once you join you're in, as far as the rest of the swarm is concerned. Re-joining is ignored. Perhaps it should be persisted but I can't think of a case where it would matter.